examples/migrations/langmem/README.md

```
@@ -0,0 +1,161 @@
+
+```bash
+pip install -e .
+pip install -r examples/migrations/langmem/requirements.txt
+cd examples/migrations/langmem
+python run.py
+```
+
+No API keys needed for this. It runs five steps and writes everything to
+`artifacts/`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)